app/src/main/java/edu/grinnell/grinnell_publications_android/Services/POJO/JStory.java

```diff
@@ -0,0 +1,110 @@
+package edu.grinnell.grinnell_publications_android.Services.POJO;
+
+
+import java.util.List;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class JStory {
+
+    @SerializedName("datePublished")
+    @Expose
+    private String datePublished;
+    @SerializedName("brief")
+    @Expose
+    private String brief;
+    @SerializedName("headerImage")
+    @Expose
+    private String headerImage;
+    @SerializedName("publication")
+    @Expose
+    private String publication;
+    @SerializedName("dateEdited")
+    @Expose
+    private String dateEdited;
+    @SerializedName("id")
+    @Expose
+    private String id;
+    @SerializedName("title")
+    @Expose
+    private String title;
+    @SerializedName("content")
+    @Expose
+    private String content;
+    @SerializedName("authors")
+    @Expose
+    private List<JAuthor> authors = null;
+
+    public String getDatePublished() {
+        return datePublished;
+    }
+
+    public void setDatePublished(String datePublished) {
+        this.datePublished = datePublished;
+    }
+
+    public String getBrief() {
+        return brief;
+    }
+
+    public void setBrief(String brief) {
+        this.brief = brief;
+    }
+
+    public String getHeaderImage() {
+        return headerImage;
+    }
+
+    public void setHeaderImage(String headerImage) {
+        this.headerImage = headerImage;
+    }
+
+    public String getPublication() {
+        return publication;
+    }
+
+    public void setPublication(String publication) {
+        this.publication = publication;
+    }
+
+    public String getDateEdited() {
+        return dateEdited;
+    }
+
+    public void setDateEdited(String dateEdited) {
+        this.dateEdited = dateEdited;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
```

Comment:
Be a bit more defensive here: Check for null ids and exit if they are (for the lack of a better way to handle this. Ideally, we should be logging this to some error tracking tool like Crashlytics). See comments on others below.